<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>672</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/search</normaloff>:/search</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>118</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QCheckBox" name="contractCheckBox">
          <property name="text">
           <string>Contract</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="contractLineEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="codCheckBox">
          <property name="text">
           <string>Cod</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="codLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="denumireCheckBox">
          <property name="text">
           <string>Denumire</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="denumireLineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="furnizorCheckBox">
          <property name="text">
           <string>Furnizor</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QComboBox" name="furnizorComboBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="findButton">
          <property name="text">
           <string>Find</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/search</normaloff>:/search</iconset>
          </property>
          <property name="shortcut">
           <string>Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>Anuleaza</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/cancel</normaloff>:/cancel</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="loadingLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string notr="true">id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Contract</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Auto</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cod</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Denumire</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cantitate</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Furnizor</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Search</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>contractCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>contractLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>27</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>codCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>codLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>denumireCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>denumireLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>78</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>furnizorCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>furnizorComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>121</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
